### DISCUSSION TITLE
Logging question!

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js-ui#2273] [General]
I want to log all traffic to/from a specific device for a few days to do some troubleshooting with zooz. Is there a better/worse way to do that with as minimal chatter/pain as possible? :) 

Should these settings do it? If so, which file does it log to in the store directory?

![image](https://user-images.githubusercontent.com/19985320/154586067-ac31235e-64e9-42d5-a4c1-aa006edbb486.png)

**Comment by robertsLando** [comment]
@Botched1 You should enable only driver logs in zwavejs section, not application logs (on general section). With this settings you will only log node `123` (dunno if maybe you was looking for 1, 2, 3 ?). Also the level is ok, even debug would be good

**Reply by Botched1** [reply]
OK. I think I got it. And I did mean node # 123. :) I have a lot of zwave nodes (115+).

Thanks!!!
